dev/ProgressBar/ProgressBar.xaml

```diff
@@ -100,162 +149,128 @@
-                                            Storyboard.TargetProperty="Fill" >
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SystemBaseMediumLowColor}" />
-                                        </ObjectAnimationUsingKeyFrames>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="DeterminateProgressBarIndicator"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="{ThemeResource ProgressBarIndicatorPauseOpacity}"
-                                            Duration="0:0:0.25" />
                                     </Storyboard>
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
-                        <Border x:Name="ProgressBarRoot"
-                            Background="{TemplateBinding Background}"
+                        <Border
+                            x:Name="ProgressBarRoot"
                             BorderBrush="{TemplateBinding BorderBrush}"
                             BorderThickness="{TemplateBinding BorderThickness}"
+                            Padding="{TemplateBinding Padding}"
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                            contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
+                            contract7NotPresent:CornerRadius="{StaticResource ProgressBarCornerRadius}">
                             
                             <Border Clip="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ClipRect}">
-                                <Grid>
-                                    <Rectangle x:Name="DeterminateProgressBarIndicator"
-                                        Margin="{TemplateBinding Padding}"
+                                <Grid Height="{TemplateBinding MinHeight}">
+                                    
+                                    <Rectangle
+                                        x:Name="ProgressBarTrack"
+                                        Fill="{TemplateBinding Background}"
+                                        Height="{StaticResource ProgressBarTrackHeight}"
+                                        Width="{TemplateBinding Width}"
+                                        VerticalAlignment="Center"
+                                        RadiusX="{Binding Source={StaticResource ProgressBarTrackCornerRadius}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
+                                        RadiusY="{Binding Source={StaticResource ProgressBarTrackCornerRadius}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}">
```

Comment:
Should these be?

contract7NotPresent:RadiusX
contract7NotPresent:RadiusY

Reply:
nope, we don't want it to be customizeable

Reply:
What I meant is I see below in DeterminateProgressBarIndicator, IndeterminateProgressBarIndicator, IndeterminateProgressBarIndicator2 for example, RadiusX is proceeded by contract7NotPresent: . Why is it different here (no "contract7NotPresent:")?

`contract7NotPresent:RadiusX="{Binding Source={StaticResource ProgressBarCornerRadius}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"`